### DISCUSSION TITLE
Is there an option to include `graphql.ts` to `index.ts`?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [dotansimha/graphql-code-generator#8460] [Q&A] [answered]
https://github.com/dotansimha/graphql-code-generator/blob/0b27412700f49849b4e15f1a30d590f9a6e5b301/examples/front-end/react/apollo-client/src/gql/index.ts#L1-L2

as outlined above, is there an option in `codegen.ts` to include `graphql.ts` to `index.ts`?

**Comment by charlypoly** [comment, marked as answer]
Hi @lifeiscontent,

There is no such option since types generated in `graphql.ts` are not meant to be used externally.
If you want to use those types directly, please prefer a configuration relying on the `typescript` and `typescript-operations` directly.
